third_party/contour-latest/net-contour.yaml

```diff
@@ -100,7 +100,7 @@ spec:
       - name: controller
         # This is the Go import path for the binary that is containerized
         # and substituted here.
-        image: gcr.io/knative-nightly/knative.dev/net-contour/cmd/controller@sha256:c276b6b4d2b21ac0455df8d2d6be7b24efea2186eaa94832f92624df4aa3956f
+        image: gcr.io/knative-nightly/knative.dev/net-contour/cmd/controller@sha256:103b910de45c55eb3f0455f0b111b36c3fbddb9b8b8a5ea00a0a6bc3c27759de
```

Comment:
might want to update the comment above to match reality?

Reply:
This is auto-generated via `ko`, so it's actually correct in source.  We can strip it from the source, but it is intended for new contributor clarity 🤷